### PR TITLE
chore(query): add system.tables query speed test

### PR DESCRIPTION
**Pull request by TCeason**
I hereby agree to the terms of the CLA available at: https://docs.databend.com/dev/policies/cla/

## Summary

1. add functions and user_functions to access white list
2. add system.tables query speed test
3. add comment filed in system.tables/columns and information_schema

- Fixes #[Link the issue here]

## Tests

- [ ] Unit Test
- [x] Logic Test
- [ ] Benchmark Test
- [ ] No Test  - _Explain why_

## Type of change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature that could cause existing functionality not to work as expected)
- [ ] Documentation Update
- [ ] Refactoring
- [ ] Performance Improvement
- [x] Other (please describe): Enhanced test capability to ensure that system tables queries do not roll back

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/14517)

### CODE REVIEW

**Comment by TCeason** [comment]
Please review this pr.

1. add functions and user_functions to access white list cc @flaneur2020
2. add system.tables query speed test cc @sundy-li 
3. add comment filed in system.tables/columns and information_schema cc @b41sh @wubx @hantmac

File: tests/suites/1_stateful/02_query/02_0006_system_table_query_speed.py [hunk 1]
```diff
@@ -11,17 +11,22 @@
...
-else:
-    print("normal")
+def test_speed(cursor, name):
+    sql = "select count() from system."+ name +" where database='test'"
+    cursor.execute(sql)
+    start = datetime.now()
+    cursor.execute(sql)
+    execute = (datetime.now()-start)
+    # in ci 0.06s, in local debug build cost 0.03s
+    if execute.total_seconds() > 0.08:
```
Comment (BohuTANG) [inline tests/suites/1_stateful/02_query/02_0006_system_table_query_speed.py:25]: We only need to ensure that it is less than a certain value, for example 1s.
The test is to check if have a performance degradation, not a benchmark.

It's important to note that the CI environment may vary, which could impact the test results.

Comment (TCeason) [reply tests/suites/1_stateful/02_query/02_0006_system_table_query_speed.py:25]: I think it's ok? In ci it will cost 0.068s.

Comment (BohuTANG) [reply tests/suites/1_stateful/02_query/02_0006_system_table_query_speed.py:25]: It may be flaky, CI env is **vary**.

Comment (TCeason) [reply tests/suites/1_stateful/02_query/02_0006_system_table_query_speed.py]: up-threshold to 0.1 second.

**Review by flaneur2020 — APPROVED** [review]

File: src/query/storages/system/src/columns_table.rs [hunk 2]
```diff
@@ -142,15 +142,42 @@ impl ColumnsTable {
...
+                if table.engine() != VIEW_ENGINE {
+                    let schema = table.schema();
+                    let field_comments = table.field_comments();
+                    let n_fields = schema.fields().len();
+                    for (idx, field) in schema.fields().iter().enumerate() {
+                        // compatibility: creating table in the old planner will not have `fields_comments`
+                        let comment = if field_comments.len() == n_fields
+                            && !field_comments[idx].is_empty()
+                        {
+                            format!("'{}'", &field_comments[idx].as_str().replace('\'', "\\'"))
```
Comment (b41sh) [inline src/query/storages/system/src/columns_table.rs:160]: how about use `format!("'{:?}'", &field_comments[idx])`?

Comment (TCeason) [reply src/query/storages/system/src/columns_table.rs:160]: > how about use `format!("'{:?}'", &field_comments[idx])`?

O I got you.

Comment (TCeason) [reply src/query/storages/system/src/columns_table.rs:161]: Note:  can not use debug print, will add double quote

**Review by b41sh — APPROVED** [review]

**Pull request merged**